### ISSUE TITLE
Framerate issue with imported video

### ISSUE THREAD

**Issue by rgov** [VIAME/VIAME#157]
At our workshop, @hsosik had uploaded a video and frames were extracted at the wrong frame rate. @BryonLewis had said the frame rate was hardcoded at 10 Hz or something like that.

**Comment by BryonLewis** [comment]
To add some more information.  The generic object detector pipe seems to be passed the proper downsampler setting but the results returned seemd to always be at 10Hz and not at the user's chosen framerate.
```
Running command: . /opt/noaa/viame/setup_viame.sh && KWIVER_DEFAULT_LOG_LEVEL=warn kwiver runner -s input:video_reader:type=vidl_ffmpeg -p /tmp/addons/extracted/configs/pipelines/detector_generic_proposals.pipe -s input:video_filename=/tmp/tmpt99mgaen/input/NESLTER_EN668_20Jul2021_014.transcoded.mp4 -s downsampler:target_frame_rate=15.0 -s detector_writer:file_name=/tmp/tmpt99mgaen/output/detector_output.csv -s track_writer:file_name=/tmp/tmpt99mgaen/output/track_output.csv
```
For reference I included the command.  You can see the `downsampler:target_frame_rate=15.0` but the returned results are not correct.
I verified this same behavior hapening outside of the docker image and on the downloaded version as well.

**Comment by rgov** [comment]
Maybe the code that actually does it is here:

https://github.com/Kitware/kwiver/blob/78992c7393a2a13a84fed476ac4e8f98dd86d814/sprokit/processes/core/downsample_process.cxx

**Comment by mattdawkins** [comment]
I believe this was fixed a fixed a couple weeks back in v0.19.6 but re-open if any datasets are still problematic
